packages/libs/react-components/src/components/Card/styles.ts

```diff
@@ -0,0 +1,84 @@
+/* eslint @kadena-dev/typedef-var: 0 */
+
+import { styled } from '../../styles';
+
+export const CardColors: string[] = [
+  'default',
+  'primary',
+  'secondary',
+  'positive',
+  'warning',
+  'negative',
+];
+
+export const StyledCard = styled('div', {
+  boxSizing: 'border-box',
+  backgroundColor: '$colors$neutral1',
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'flex-start',
+  padding: '24px',
+  minWidth: '546px',
+  width: 'min-content',
+  height: 'min-content',
+  boxShadow: '0px 0px 10px rgba(0, 0, 0, 0.04)',
+  borderRadius: '8px',
```

Comment:
We generally want to avoid unique values to certain properties because we want all of these to be consistent and align with the design system. To achieve this, we add values like border radius to the theme so we can reuse the same values via a token. If this value isn't already in the theme, you should verify with @isaozler if it was intentional to use a new style, or if it needs to be integrated into the theme for reuse